back-end/server/api/users.js

```diff
@@ -62,15 +62,15 @@ router.get('/:userId/friends/:friendId', async (req,res,next) => {
   }
 })
 
-router.get('/:userId/friends/:friendId/activities', async (req,res,next) => {
-  try{
+router.get('/:userId/friends/:friendId/activities', async (req, res, next) => {
```

Comment:
Did you miss changing some of the routes in this file? some of the routes still say friends